build-tools/create-packs/Directory.Build.targets

```diff
@@ -126,7 +126,9 @@
         DestinationFolder="$(DotNetPreviewPath)packs\$([System.String]::Copy('%(_WLPacks.Filename)').Replace('.$(_WLPackVersion)', ''))\$(_WLPackVersion)"
     />
     <MakeDir Directories="$(DotNetPreviewPath)template-packs" />
-    <Copy SourceFiles="@(_WLTemplates)" DestinationFolder="$(DotNetPreviewPath)template-packs" />
+    <!-- Copy template packs to lowercase destination when running on Linux -->
+    <Copy Condition="'$(HostOS)' != 'Linux'" SourceFiles="@(_WLTemplates)" DestinationFolder="$(DotNetPreviewPath)template-packs" />
+    <Copy Condition="'$(HostOS)' == 'Linux'" SourceFiles="@(_WLTemplates)" DestinationFiles="@(_WLTemplates->'%(Filename)'->ToLower()->'$(DotNetPreviewPath)template-packs\%(Identity)')" />
```

Comment:
I think we could always do the lower case, even the .NET 5 ones are lower:

![image](https://user-images.githubusercontent.com/840039/116122669-b86ed880-a687-11eb-919a-87a4643aa944.png)

Reply:
Sounds good, we can always copy to a lowercase location. I am also hoping to wait for a response on the issue or to chat about this quickly on Thursday to see if we should change the template pack ID / version (here and in macios) instead. If there won't be SDK changes coming that will process the packs regardless of casing then we should probably make that change instead.

Reply:
Could someone explain that `DestinationFiles` expression to me?  I've never seen *multiple* `->` constructs before.  Does the rightmost `%(Identity)` apply to the "intermediate" list resulting from `@(_WLTemplates->'%(Filename)'->ToLower())`, or is it `%(_WLTemplates.Identity)`?

Reply:
The `%(Identity)` in this case will be the product of the transform plus the item function, so the value of
  `@(_WLTemplates->'%(Filename)'->ToLower())` best summarizes it.